<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MortageWindow</class>
 <widget class="QWidget" name="MortageWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1270</width>
    <height>630</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="backgroundLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-10</y>
     <width>1280</width>
     <height>640</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>82</y>
     <width>1101</width>
     <height>491</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	background-color: #151c26;
	border-radius: 20px;
	border: 1px solid #44eb9b;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: #44eb9b;
	color: #151c26;
	border-radius: 10px;
	border-image: url(:/src/image/previous.png) stretch; 
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>140</y>
     <width>351</width>
     <height>201</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <widget class="QLabel" name="label_4">
      <property name="styleSheet">
       <string notr="true">QLabel {
	color: #44eb9b;
	font-weight: bold;
	font-size: 17px;
}</string>
      </property>
      <property name="text">
       <string>Interest rate</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_term">
      <property name="styleSheet">
       <string notr="true">QLineEdit {
	color: #44eb9b;
	background-color: rgb(74, 71, 169);
}</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="styleSheet">
       <string notr="true">QLabel {
	color: #44eb9b;
	font-weight: bold;
	font-size: 17px;
}</string>
      </property>
      <property name="text">
       <string>Loan amount</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QRadioButton" name="differentiatedButton">
      <property name="styleSheet">
       <string notr="true">QRadioButton{
	color: #44eb9b;
	font-weight: bold;
	font-size: 17px;
}</string>
      </property>
      <property name="text">
       <string>Differentiated</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QRadioButton" name="annuityButton">
      <property name="styleSheet">
       <string notr="true">QRadioButton{
	color: #44eb9b;
	font-weight: bold;
	font-size: 17px;
}</string>
      </property>
      <property name="text">
       <string>Annuity</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit_rate">
      <property name="styleSheet">
       <string notr="true">QLineEdit {
	color: #44eb9b;
	background-color: rgb(74, 71, 169);
}</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_3">
      <property name="styleSheet">
       <string notr="true">QLabel {
	color: #44eb9b;
	font-weight: bold;
	font-size: 17px;
}</string>
      </property>
      <property name="text">
       <string>Term</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_amount">
      <property name="styleSheet">
       <string notr="true">QLineEdit {
	color: #44eb9b;
	background-color: rgb(74, 71, 169);
}</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>796</x>
     <y>510</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: #44eb9b;
	color: #151c26;
	border-radius: 20px;
}
</string>
   </property>
   <property name="text">
    <string>Calculate</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>358</y>
     <width>351</width>
     <height>2</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	background-color: #44eb9b;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCustomPlot" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>659</x>
     <y>149</y>
     <width>441</width>
     <height>331</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCustomPlot {
	background-color: #44eb9b;
	color: #151c26;
}
</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>380</y>
     <width>351</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="label_mounth_payment">
      <property name="styleSheet">
       <string notr="true">QLabel {
	color: #44eb9b;
	font-weight: bold;
	font-size: 17px;
}</string>
      </property>
      <property name="text">
       <string>Mounth payment</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_mounth_pay">
      <property name="styleSheet">
       <string notr="true">QLineEdit {
	color: #44eb9b;
	background-color: rgb(74, 71, 169);
}</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_over_payment">
      <property name="styleSheet">
       <string notr="true">QLabel {
	color: #44eb9b;
	font-weight: bold;
	font-size: 17px;
}</string>
      </property>
      <property name="text">
       <string>Over payment</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_over_pay">
      <property name="styleSheet">
       <string notr="true">QLineEdit {
	color: #44eb9b;
	background-color: rgb(74, 71, 169);
}</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_whole_payment">
      <property name="styleSheet">
       <string notr="true">QLabel {
	color: #44eb9b;
	font-weight: bold;
	font-size: 17px;
}</string>
      </property>
      <property name="text">
       <string>Whole payment</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit_whole_pay">
      <property name="styleSheet">
       <string notr="true">QLineEdit {
	color: #44eb9b;
	background-color: rgb(74, 71, 169);
}</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
